dashboard/config/locales/en.yml

```diff
@@ -1427,7 +1427,7 @@ en:
       form_instructions_3: "Not seeing your LMS in our list, or running into issues with your installation? Check out our <a href=''>guides</a>, or <a href='https://code.org/contact'>contact us</a> for additional help"
       guide_instructions: "Full guides for installing and our LTI integration <a href=''>can be found here.</a> Running into issues? Feel free to <a href=''>contact us</a> for additional support."
       mailer_confirmation_subject: "Code.org has completed your LMS registration request"
-      mailer_confirmation_body: "Great news! We’ve registered your LMS with Code.org. You should now be able to use our Code.org LMS integrations. Please feel free to review & share our <a href=''>LMS integration guides</a>, or browse our <a href=''>course offerings</a>."
+      mailer_confirmation_body: "Great news! We’ve registered your LMS with Code.org. You should now be able to use our Code.org LMS integrations. Please feel free to review & share our [LMS integration guides](%{integration_guide_url}), or browse our [course offerings](%{course_offering_url})."
```

Comment:
Good thing you are using interpolation for links because I don't think we are redacting them in this file.